.rubocop.yml

```diff
@@ -0,0 +1,37 @@
+require: 
+  - rubocop-rails
+
+AllCops:
+  NewCops: enable
+  Exclude:
+    - 'db/**/*'
+    - 'bin/**/*'
+    - 'Gemfile'
+    - 'Rakefile'
+    - 'spec/spec_helper.rb'
+    - 'node_modules/**/*'
+    - 'tmp/*'
+    - 'config/initializers/**/*'
+    - 'lib/tasks/*'
+    - 'Guardfile'
+
+Layout/LineLength:
+  Max: 100
+  Exclude: 
+    - 'config/routes.rb'
+
+Metrics/BlockLength:
+  Exclude:
+    - config/**/*
+    - spec/**/*
+
+Lint/AmbiguousBlockAssociation:
+  Exclude:
+    - spec/**/*
```

Comment:
This one i'm not as familiar with and am curious what about specs makes them not conform?

Reply:
I will have to pull that one out and see what it adds in offenses. I am happy to change it.